lib/util/exec/env.ts

```diff
@@ -39,6 +39,7 @@ const basicEnvVars = [
   'COREPACK_ROOT',
 ];
 
+// should I use the combined env here as well ?
```

Comment:
I'm not sure, maybe? We need to exclude `process.env` if `exposeAllEnv!==true`

Reply:
I think we should use it here as well